backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/UpdateUserProfilePropertyCommandTest.java

```diff
@@ -32,6 +34,9 @@ class UpdateUserProfilePropertyCommandTest extends BaseCommandTest {
     @Mock
     private DbUserDao userDaoMock;
 
+    @Mock
+    private Predicate<DbUser> isSystemSuperUserPredicate;
```

Comment:
The test condition `when(isSystemSuperUserPredicate.test(....).thenReturn(true)` seems missing for `isSystemSuperUserPredicate`

Reply:
As above: we have no super user scenario here.